### DISCUSSION TITLE
How to build  DocumentBuilder?

### DISCUSSION THREAD

**Discussion by fuchsia74** [ONLYOFFICE/DocumentBuilder#96] [Q&A] [answered]
Hi,
I would like to ask how to compile DocumentBuilder git code separately. I tried to compile using "make all" or "make", but there are errors as follows:
![image](https://user-images.githubusercontent.com/10117683/216213237-1182a2cd-0621-4ff2-b99b-584f47f6b041.png)
and
![image](https://user-images.githubusercontent.com/10117683/216213310-7507b865-f2c0-4dd2-a990-0545ba760b53.png)

Could you guide me on how to compile separately?

**Comment by Nickwiz** [comment, marked as answer]
Check out: https://github.com/ONLYOFFICE/build_tools/

**Comment by trofim24** [comment]
Yes, you can build DocumentBuilder and other of our products using [build_tools](https://github.com/ONLYOFFICE/build_tools/#document-builder)
